src/app/core/services/tracking.service.ts

```
@@ -374,6 +374,11 @@
     this.eventTrack('Sync Error', properties);
   }
 
+  // capture receipt flow, patch expenses error
+  patchExpensesError(properties: { label: Error }): void {
+    this.eventTrack('Patch expense error - capture receipt flow', properties);
+  }
```

ESLint

[error] Member patchExpensesError should be declared before all private instance method definitions.

(@typescript-eslint/member-ordering)